src/createNumbersGenerator.js

```diff
@@ -25,8 +25,13 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers = [], min = -Infinity, max = Infinity) {
+  let counter = 0;
+
+  return function(resultNumbers = numbers) {
```

Comment:
better to move filtering to the closure, you don't need to do filtering every time you call inner function